extensions/src/test/java/com/tomtom/kotlin/extensions/IfExtensionsTest.kt

```diff
@@ -100,7 +100,6 @@ internal class IfExtensionsTest {
     fun `ifNull on null`() {
         // GIVEN
         val sut: Int? = null
-        // TODO: replace with `spyk({ 1 })` when https://github.com/mockk/mockk/issues/1033 is fixed.
         val ifNullBlock: () -> Int = spyk {
             every { this@spyk.invoke() } answers { 1 }
         }
```

Comment:
I meant that you can *resolve* the TODOs, not just remove them.
For example, this block can be changed into (as it used to be):
```suggestion
        val ifNullBlock = spyk({ 1 })
```

Also, there are more TODOs in this file than the ones you removed.

Reply:
Resolved

Reply:
The other TODOs were still not addressed, I figured it's faster to push a commit with what I was requesting. :)

Reply:
Thanks